src/components/SelectCol.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+class SelectCol extends React.Component {
+    constructor( props ) {
+        super( props );
+        this.getCheckedValues = this.getCheckedValues.bind(this);
+        this.state = {
+            languages: [ "Arabic", "Serbian", "Croatian", "Russian", "German", "Hebrew", "French",
+                "Hungarian", "Slovak", "Spanish", "Portugues", "Turkce", "Greek", "Romanian"],
+            defaultLans: this.props.default
+        }
+    }
+
+    getCheckedValues( index ) { //Get the selected Value and pass to other component
+        //toggle the default languages
+        if (index != null) {
+            const newLans = this.state.defaultLans.slice() //copy the array
+            newLans[ index ].value = !this.state.defaultLans[ index ].value;  //execute the manipulations
+            this.setState({ defaultLans: newLans }) //set the new state
+        }
+
+        const values = [ ];
+        const boxes = document.getElementsByClassName( "custom-control-input" );
+        for(let i=0; i< boxes.length; i++){
+            if(boxes[i ].checked){
+               values.push(boxes[ i ].id);
+            }
```

Comment:
space for line 24-26.
for ( let i = 0; i < boxes.length; i++ ) {
    if ( boxes[ i ].checked ) {
        values.push( boxes[ i ].id )  
    }
}